internal/orchestrator/notify/slack.go

```diff
@@ -44,7 +44,7 @@ func (ss *slackSender) Send(upd *v1types.ConditionUpdateEvent) error {
 		"conditionID": upd.ConditionUpdate.ConditionID.String(),
 	})
 
-	le.Debug("sending slack notification")
+	le.Trace("sending slack notification")
```

Comment:
Why make this change? We log at `Info` in production?

Reply:
Since each status update creates a log entry, this was too verbose for debugging with. We can come with some guidelines separately on what should be at debug vs trace.

Reply:
Guidelines for when to use debug, info, or trace are basically impossible to follow. Just have to take it case by case IMO. This is definitely a Trace IMO.